cloud_AWS/terraform/module/examples/multiple-vpc-regions/README.md

```diff
@@ -0,0 +1,75 @@
+# Multiple VPC's from different regions
+
+The configuration in this directory creates a configuration for each VPC ID located in the input_data.json file.
+
+
+## Requirements
+
+- Installed and configured AWS CLI
+    - [installation](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html)
+    - [configuration](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html)
+- Created VPC
```

Comment:
VPCs are created by default in all AWS regions, so this requirement is always fulfilled

Reply:
What if someone deleted this default VPC? Maby I'll change it to "Created VPC (or default VPC)"

Reply:
OK